js/app/src/Blocks.svelte

```diff
@@ -417,6 +418,19 @@
 					handled_dependencies[i].push(id);
 				});
 		});
+		if (!attached_error_listeners) {
```

Comment:
Apparently `handle_mount` runs multiple times, so I had to use attached_error_listeners to make sure I only attach error listeners once. Could not figure out why handle mount runs multiple times.

Reply:
That's by design, because `onMount` from child components will run at different times with different elements being ready at different points. So we need to allow this onMount callback to rerun as many times as is necessary to ensure that we handle the mounting of all child components, that said I wanna reword this when I refactor this code shortly.

Reply:
So onMount runs every time a child is added?

Reply:
ok, did something similar

Reply:
attached_error_listeners is an array of ids now

Reply:
Will re-review.